stack.yaml

```diff
@@ -0,0 +1,2 @@
+description: Stack that uses EKS as base
```

Comment:
If you think the text makes more sense:

`description: Stack to provisioning AWS EKS cluster infrastructure`